apps/microtvm/reference-vm/base-box-tool.py

```diff
@@ -358,14 +358,18 @@ def test_command(args):
 
 
 def release_command(args):
+    vm_name = f"mehrdadh/microtvm-{args.platform}"
+    if args.platform == "zephyr":
+        vm_name = f"{vm_name}-{args.zephyr_version}"
```

Comment:
can you ensure `args.zephyr_version` is not `None`, else it will release microtvm-None, I think?

Reply:
thanks for catching this!

Reply:
I think it would be better to not by default publish any version, but instead require that the user explicitly state it. otherwise, we'll wind up publishing lots of zephyr-2.5 boxes :)

also--we could probably call this platform-version, since `tlcpack/microtvm-<platform>-<platform_version>` is a pretty generic format we'll keep if we add more platforms

Reply:
I changed it to platform_version and rework the input args with sub_parser.